src/Chapter1.hs

```diff
@@ -437,7 +437,7 @@ Implement the function that takes an integer value and returns the next 'Int'.
   function body with the proper implementation.
 -}
 next :: Int -> Int
-next x = error "next: not implemented!"
+next x = x+1
```

Comment:
Great one!
In Haskell  all functions and operators are separated from arguments by spaces 👾 :
```suggestion
next x = x + 1
```